app/src/main/java/com/example/candyshop/Screen.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.candyshop
+
+sealed class Screen(val route: String) {
+    object MainScreen : Screen("main_screen")
+    object DetailsScreen : Screen("details_screen")
+
+    fun withArgs(vararg args: String): String {
```

Comment:
you can just write:

fun withArgs(vararg args: String): String = buildString {
            append(route)
            args.forEach { arg -> append("/$arg") }
        }

no need to use return